Development/src/pages/senders.js

```diff
@@ -373,7 +376,17 @@ const ShowActiveTab = ({ controllerProps, ...props }) => {
         >
             <SimpleShowLayout>
                 <TextField label="ID" source="id" />
-                <TextField label="Receiver ID" source="$active.receiver_id" />
+                {get(controllerProps.record, '$active.receiver_id') && (
+                    <ReferenceField
+                        basePath="/senders"
```

Comment:
This looks like a copy&paste error, as didn't it ought to be linking to `/receivers`?